client/components/overviewWidget/ProductInformation.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import StarAverage from '../shared/StarAverage.jsx';
+// import axios from 'axios';
+
+class ProductInformation extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      product: null,
+      reviewMetaData: null,
+      reviewAverage: null,
+      reviewCount: 0
+    };
+  }
+
+  componentDidMount() {}
+
+  componentDidUpdate(prevProps) {
+    if (this.props.product !== prevProps.product) {
```

Comment:
I think if you don't save product in state then this become un-necessary since the updated products props and all the other props should get passed down